Sources/_InternalTestSupport/XCTSkipHelpers.swift

```diff
@@ -17,6 +17,18 @@ import XCTest
 import class Basics.AsyncProcess
 import struct TSCBasic.StringError
 
+public func skipOnWindowsAsTestCurrentlyFails(because reason: String? = nil) throws {
+    #if os(Windows)
+    let failureCause: String
+    if reason == nil {
```

Comment:
The idiomatic way to do this in Swift would be:

```swift
if let reason {
    failureCause = " because \(reason.description)"
} else {
    failureCause = ""
}
```

(you want to avoid the force unwrap)

Reply:
Thanks @jakepetroules .. I knew there was a more idiomatic way but my internet search failed me :)